### PR TITLE
[Peggy] Feature/peggy test coverage

### PR DESCRIPTION
**Pull request by juniuszhou**
Add two python scripts for test the peggy functionalities, also could be used to do smoking test after the deployment. Use the python to write test flow also be better to add more test cases in the future.

### CODE REVIEW

**Review by ElliotFriedman — APPROVED** [review]

File: scripts/README.md [hunk 1]
```diff
@@ -11,3 +11,13 @@ Scripts for local development and testing.
 ```
 ./scripts/setup-e2e.sh
 ```
+
```
Comment (khdegraaf) [inline scripts/README.md:14]: This is an inaccurate or incomplete set of steps to run the tests.  I can get the python scripts to run if I run the manual steps in peggy-e2e-test.md at the beginning.  Although it took me two tries.  Rrunning peggy-basic-test.py worked, but then running peggy-e2e-test.py after failed due to 

> 2020/11/04 09:13:18 VM Exception while processing transaction: revert Not enough locked assets to complete the proposed prophecy
>calvins-mbp:testnet-contracts kevindegraaf$ ebrelayer init tcp://localhost:26657 ws://localhost:7545/ 0x30753E4A8aad7F8597332E8137

but worked if I ran it again.  Should fix ebrelayer to not crash here?  In any event, documentation and script should work easily and automatically.  Goal is for this to be turned on as a true smoke test that could be run in Github Actions check.

**Review by khdegraaf — CHANGES_REQUESTED** [review]
This needs to work automatically from script, and it isn't quite there yet.

File: scripts/README.md (hunk 1 above)
Comment (juniuszhou) [reply scripts/README.md:20]: I will add description for test environment for local node and testnet. For the relayer crash problem, I will create an issue in github and not deal with it in this pr.

Comment (khdegraaf) [reply scripts/README.md:20]: OK

**Review by khdegraaf — APPROVED** [review]

File: scripts/README.md (hunk 1 above)
Comment (khdegraaf) [reply scripts/README.md:20]: Should try to fully automate in Github Actions at some point too.

**Pull request merged**